project_admin/templates/oh_proj_management/base.html

```diff
@@ -0,0 +1,69 @@
+{% load static %}
```

Comment:
(This comment is related to this whole file, not this particular first line.)

I'd rather see this file to have been created by moving the original file to this location and name (using `git mv`) not making a copy of it. The old location should be obsolete (replaced by the new location), and by using `git mv` you retain authorship history for items within the file.